addons/mail/static/src/models/message/message.js

```diff
@@ -448,6 +486,11 @@ function factory(dependencies) {
     }
 
     Message.fields = {
+        actionList: one2one('mail.message_action_list', {
```

Comment:
This is potentially problematic as a "component field" because there can be the same message model shown through different components at the same, and they each will share this value.

We are currently lacking a "message component" model so not your fault, but to keep in mind.

I think in this case it might be ok?
Otherwise the fix is to create one "message component" equivalent model for each message in thread view, but if there is no functional issue here we will do it later.

Reply:
There is no functional issue, this model is only used to store state regarding whether the delete confirmation dialog is open or not, when that is the case you cannot really navigate to another place where the message will be rendered and should have an action list in a different state so I think it's fine for now. Should be moved to the MessageView model once it is implemented though.